extensions/docbook-snippets/README.md

```diff
@@ -0,0 +1,65 @@
+# docbook-snippets README
```

Comment:
Was that file autogenerated? I assume it is as it sounds very generic. You probably want to adapt the following subsections. :wink: